src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -341,10 +353,68 @@ public function testLegacyAsText()
      */
     public function testLegacyAsXml()
     {
-        $command = new \TestCommand();
+        $command = new TestCommand();
         $command->setApplication(new Application());
         $tester = new CommandTester($command);
         $tester->execute(array('command' => $command->getName()));
         $this->assertXmlStringEqualsXmlFile(self::$fixturesPath.'/command_asxml.txt', $command->asXml(), '->asXml() returns an XML representation of the command');
     }
+
+    public function testSetProcessTitle()
+    {
+        $command = new TestCommand();
+
+        $this->assertNull(\PHPUnit_Framework_Assert::getObjectAttribute($command, 'processTitle'));
+
+        $command->setProcessTitle('process title');
+        $inputMock = $this->prophesize('Symfony\Component\Console\Input\InputInterface');
+        $outputMock = $this->prophesize('Symfony\Component\Console\Output\OutputInterface');
+        $command->run($inputMock->reveal(), $outputMock->reveal());
+
+        $this->assertSame('process title', \PHPUnit_Framework_Assert::getObjectAttribute($command, 'processTitle'));
+    }
+
+    /**
+     * @dataProvider getSetAliasesData()
+     *
+     * @param string[]|\Traversable $aliases
+     */
+    public function testSetAliases($aliases)
+    {
+        $command = new TestCommand();
+        $command->setAliases($aliases);
+
+        $this->assertEquals($aliases, $command->getAliases());
+    }
+
+    /**
+     * @return array[]
+     */
+    public function getSetAliasesData()
```

Comment:
world peace would make me happy :smile: 
But regarding PR:
How about `getAliasesData` or `getDataForSettingAliases` or whatever, but not `getSet... go!` ;)